jax/_src/config.py

```diff
@@ -818,6 +818,14 @@ def update_thread_local_jit_state(**kw):
           'option is set, the log level is WARNING; otherwise the level is '
           'DEBUG.'))
 
+explain_cache_misses = define_bool_state(
```

Comment:
Maybe this should be an enum with verbosity levels?

Reply:
What would the verbosity levels control? Or do you just mean for future-proofing?

Reply:
I guess we can do it in the future. 

I was talking about the verbosity level where we print everything out (miss and hit explanations).

Reply:
If we think we will ever make this an enum, it might be good to make it an enum now, so we don't break existing code. So I'm open to making it an enum but what should the options be?

Reply:
enum_values=['log_miss', 'log_everything'] with default being log_miss?

(Even though we haven't done the log_everything bit yet)

Do you think it's a good idea to print everything?